docs/sdks/server-sdks/node.md

```diff
@@ -330,30 +330,207 @@ To read more about different flag types, see the page on [Flag Variations](/feat
 
 How the SDK fetches experiment configurations is configurable via additional optional initialization options:
 
-| Option | Description | Default |
-| ------ | ----------- | ------- | 
-| **`requestTimeoutMs`** (number) | Timeout in milliseconds for HTTPS requests for the experiment configurations. | `5000` |
-| **`numInitialRequestRetries`** (number) | Number of _additional_ times the initial configurations request will be attempted if it fails. This is the request typically synchronously waited (via `await`) for completion. A small wait will be done between requests. | `1` |
-| **`pollAfterFailedInitialization`** (boolean) | Poll for new configurations even if the initial configurations request failed. | `false` |
-| **`throwOnFailedInitialization`** (boolean) | Throw an error (reject the promise) if unable to fetch initial configurations during initialization. | `true` |
-| **`numPollRequestRetries`** (number) | If polling for updated configurations after initialization, the number of additional times a request will be attempted before giving up. Subsequent attempts are done using an exponential backoff. | `7` |
+| Option                                        | Description                                                                                                                                                                                                                 | Default |
```

Comment:
Not many content changes here or the next table. My IDE suggested moving the pipes so they all line up 🤷‍♂️ 🧹 

(Except for the content changes I explicitly commented on)